app/scripts/controllers/rif/rns/resolver/index.js

```diff
@@ -136,10 +137,16 @@ export default class RnsResolver extends RnsJsDelegate {
    * @param domainName DomainName with the .rsk extension
    * @returns {Promise<unknown>}
    */
-  getChainAddressForResolvers (domainName) {
+  getChainAddressForResolvers (domainName, subdomain = '') {
     return new Promise((resolve, reject) => {
-      const addrChangedEvent = this.multiChainresolverContractInstance.AddrChanged({node: namehash.hash(domainName)}, {fromBlock: 0, toBlock: 'latest'});
-      const chainAddrChangedEvent = this.multiChainresolverContractInstance.ChainAddrChanged({node: namehash.hash(domainName)}, {fromBlock: 0, toBlock: 'latest'});
+      let node = namehash.hash(domainName);
+      if (subdomain && subdomain !== '') {
```

Comment:
I don't get this check, you check if the subdomain is different from a single point? why? can you explain that?